CityOfDetroit/bloom
# 1348/partners listing search

# Pull Request Template

## Issue Overview

This PR addresses #1348


## Description

This re-implements search on Detroit Partner's and refactoring it to utilize the new AG table and AG Pagination components. 
## How Can This Be Tested/Reviewed?

This PR can be tested by going to Detroit Partners, logging in, and typing in the search bar (at least 3 characters). The listings should automatically filter to match that string. Additionally, test all other existing partners table features. This includes sorting by column, ordering the columns, and logging out and logging back in to see your most recent table configuration. 
  
## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
